src/Controller/InspectController.php

```diff
@@ -0,0 +1,109 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Yiisoft\Yii\Debug\Api\Controller;
+
+use InvalidArgumentException;
+use Closure;
+use Psr\Container\ContainerInterface;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use ReflectionClass;
+use Yiisoft\Config\ConfigInterface;
+use Yiisoft\DataResponse\DataResponseFactoryInterface;
+use Yiisoft\VarDumper\VarDumper;
+use Yiisoft\Yii\Debug\Api\ApplicationState;
+use Yiisoft\Yii\Debug\Api\PhpUnitCommand;
+
+class InspectController
+{
+    public function __construct(
+        private DataResponseFactoryInterface $responseFactory,
+    ) {
+    }
+
+    public function config(ContainerInterface $container): ResponseInterface
+    {
+        $config = $container->get(ConfigInterface::class);
+
+        // TODO: pass different envs
```

Comment:
???

Reply:
It's technically possible to fetch `console` group. But now it's unnecessary.

Reply:
I've added it, but `$request['group']` doesn't exist now.